### PR TITLE
Updates from Dr. Stevanovic

### PR DESCRIPTION
**Pull request by wkozmaNTIA**
Incorporates pull request #2  with a few minor edits for an updated release

### CODE REVIEW

**Review by RBallardNTIA — APPROVED** [review]
Just a few comments about adding citations or clarifying comments/comment readability, mostly w.r.t. the old Fortran comments carried over.

File: src/FlatEarthCurveCorrection.cpp [hunk 1]
```diff
@@ -34,14 +35,47 @@ double FlatEarthCurveCorrection(complex<double> Delta, complex<double> q, double
...
+        A[0] = 1.0;
+        A[1] = -j * sqrt(PI);
+        A[2] = -2.0;
+        A[3] = j * sqrt(PI) * (1.0 + 1.0/(4.0*q3));
+        A[4] = 4.0/3.0 * ( 1.0 + 1.0/(2.0*q3) );
+        A[5] = -j * sqrt(PI) / 4.0 * (1.0 + 3.0 / (4.0 * q3));
+        A[6] = -8.0/15.0 * ( 1.0 + 1.0/q3 + 7.0 / (32.0 * q6) );
+        A[7] = j * sqrt(PI) / 6.0 * ( 1.0 + 5.0/(4.0*q3) + 27.0 /( 32.0*q6 ) );
+        A[8] = 16.0/105.0 * ( 1.0 + 3.0/(2.0*q3) + 27.0/(32.0*q6) );
+        A[9] = -j * sqrt(PI)/24.0 * (1.0 + 7.0/ (4.0*q3) + 5.0/(4.0*q6) + 21.0/(64.0*q9));
```
[the remark this answers is not in the document]
Comment (wkozmaNTIA) [reply src/FlatEarthCurveCorrection.cpp:66]: Citation added.

File: src/wofz.cpp [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+ 
+double XABSQ = pow(XABS,2.0);
+double XQUAD = XABSQ - pow(YABS, 2.0);
+double YQUAD = 2*XABS*YABS;
+ 
+bool A     = (QRHO < 0.085264E0);
+ 
+if (A) {
+    
+    // IF (QRHO.LT.0.085264D0) THEN THE FADDEEVA-FUNCTION IS EVALUATED
```
[the remark this answers is not in the document]
Comment (wkozmaNTIA) [reply src/wofz.cpp:93]: Come on, its FORTRAN code :)
.LT. is <
D0 just means decimal with no exponential notation portion

Fixed.

File: src/wofz.cpp [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    V    = U1*V2 + V1*U2;
+    
+} else {
+    
+    // IF (QRHO.GT.1.O) THEN W(Z) IS EVALUATED USING THE LAPLACE
+    // CONTINUED FRACTION
+    // NU IS THE MINIMUM NUMBER OF TERMS NEEDED TO OBTAIN THE REQUIRED
+    // ACCURACY
+    
+    // IF ((QRHO.GT.0.085264D0).AND.(QRHO.LT.1.0)) THEN W(Z) IS EVALUATED
```
[the remark this answers is not in the document]
Comment (wkozmaNTIA) [reply src/wofz.cpp:126]: Updated.

File: src/wofz.cpp [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+//             THE VALUES OF THIS VARIABLE HAVE THE FOLLOWING
+//             MEANING :
+//             FLAG=.FALSE. : NO ERROR CONDITION
+//             FLAG=.TRUE.  : OVERFLOW WILL OCCUR, THE ROUTINE
+//                            BECOMES INACTIVE
+//
+// FURTHERMORE THE PARAMETER FACTOR EQUALS 2/SQRT(PI)
+//
+// THE ROUTINE IS NOT UNDERFLOW-PROTECTED BUT ANY VARIABLE CAN BE
+// PUT TO 0 UPON UNDERFLOW;
```
[the remark this answers is not in the document]
Comment (wkozmaNTIA) [reply src/wofz.cpp:44]: Per discussion and testing with folks, we are safe for how this method is used.  As the valid input parameter range within LFMF will avoid this occurring.

**Pull request merged**